Add long covid

rose-higgins opened the pull request.

rose-higgins committed:
Add long COVID codelists

rose-higgins committed:
Add long COVID to study definition

rose-higgins committed:
Add long COVID to intermediate data processing steps

rose-higgins committed:
Add long COVID to coverage report

rose-higgins committed:
Merge branch 'report' of https://github.com/opensafely/antibody-and-antiviral-deployment into Add_long_covid

rose-higgins committed:
Fix codelist issue

rose-higgins committed:
Include long covid in final report RMD

rose-higgins committed:
Update file paths for server running

milanwiedemann reviewed (approved):
I cant knot the report `mabs_and_antivirals_coverage_report.Rmd` without errors. Unfortunately I get the same error message that we got when you ran it on your computer – and I dont understand what the problem is.

```R
Error in `include_graphics()`:
! Cannot find the file(s): "_wip/figures/figure_cum_eligiblity.png"
Backtrace:
 1. knitr::include_graphics(fs::path(output_dir_rmd, "figures", "figure_cum_eligiblity.png"))

Execution halted
```

But the general setup to have a work in progress report locally looks good! I think you added some paths to the updated `.gitignore` file that wont make a difference (e.g., `reports/coverage/mabs_and_antivirals_coverage_report.html` as it's already known to git).

I think the main ones you need are the following, although I have to admit that I'm struggeling to fully understand everything that's going on in this repo.

```
released_outputs/reports_wip/coverage/*
reports/coverage_wip/*
```

rose-higgins commented:
> I cant knot the report `mabs_and_antivirals_coverage_report.Rmd` without errors. Unfortunately I get the same error message that we got when you ran it on your computer – and I dont understand what the problem is.
> 
> ```r
> Error in `include_graphics()`:
> ! Cannot find the file(s): "_wip/figures/figure_cum_eligiblity.png"
> Backtrace:
>  1. knitr::include_graphics(fs::path(output_dir_rmd, "figures", "figure_cum_eligiblity.png"))
> 
> Execution halted
> ```
> 
> But the general setup to have a work in progress report locally looks good! I think you added some paths to the updated `.gitignore` file that wont make a difference (e.g., `reports/coverage/mabs_and_antivirals_coverage_report.html` as it's already known to git).
> 
> I think the main ones you need are the following, although I have to admit that I'm struggeling to fully understand everything that's going on in this repo.
> 
> ```
> released_outputs/reports_wip/coverage/*
> reports/coverage_wip/*
> ```

Thanks Milan, I actually forgot to remove the "_wip" file naming for the final PR, will add that in now, and should hopefully work as a full knit for you.

rose-higgins commented:
> I cant knot the report `mabs_and_antivirals_coverage_report.Rmd` without errors. Unfortunately I get the same error message that we got when you ran it on your computer – and I dont understand what the problem is.
> 
> ```r
> Error in `include_graphics()`:
> ! Cannot find the file(s): "_wip/figures/figure_cum_eligiblity.png"
> Backtrace:
>  1. knitr::include_graphics(fs::path(output_dir_rmd, "figures", "figure_cum_eligiblity.png"))
> 
> Execution halted
> ```
> 
> But the general setup to have a work in progress report locally looks good! I think you added some paths to the updated `.gitignore` file that wont make a difference (e.g., `reports/coverage/mabs_and_antivirals_coverage_report.html` as it's already known to git).
> 
> I think the main ones you need are the following, although I have to admit that I'm struggeling to fully understand everything that's going on in this repo.
> 
> ```
> released_outputs/reports_wip/coverage/*
> reports/coverage_wip/*
> ```

Thanks Milan, i think perhaps i updated the file names after you checked in a final commit. They switches the file usage in the report away from the "_wip" files we added before. Should work fine for knitting through successfully now.
